daemon/cmd/main.go

```diff
@@ -301,10 +306,17 @@ func main() {
 
 							client := rpc.NewRPCClient("localhost", grpcPort)
 
+							maxRoutingFee := cmd.Float("max-routing-fee")
```

Comment:
I would add `Percent` To the command itself for clarity

Reply:
will add it to the grpc call as well

Reply:
done